packages/database/src/changeHandlers/TaskCompletionHandler.js

```diff
@@ -57,23 +57,37 @@ export class TaskCompletionHandler extends ChangeHandler {
       },
     });
 
-    return tasks.map(task => task.id);
```

Comment:
I think that it would still be better to return here and then do the `map the task ids to the survey response ids` in the body of the `handleChanges`. Then you would only have to iterate once time on an array instead of on the array and then the new object so it would be simpler to understand.